src/main/java/edu/hawaii/its/api/controller/OotbRestController.java

```diff
@@ -0,0 +1,58 @@
+package edu.hawaii.its.api.controller;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.owasp.html.PolicyFactory;
+import org.owasp.html.Sanitizers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.hawaii.its.api.service.HttpRequestService;
+import edu.hawaii.its.groupings.access.UserContextService;
+import edu.hawaii.its.groupings.configuration.OotbStaticUserAuthenticationFilter;
+
+@RestController
+@RequestMapping("/api/groupings/ootb")
+public class OotbRestController {
+    private static final Log logger = LogFactory.getLog(OotbRestController.class);
+    private final PolicyFactory policy;
+
+    @Value("${url.api.2.1.base}")
+    private String API_2_1_BASE;
+
+    @Autowired
+    private UserContextService userContextService;
+
+    @Autowired
+    private OotbStaticUserAuthenticationFilter ootbAuthenticationFilter;
+
+    @Autowired
+    private HttpRequestService httpRequestService;
+
+    // Constructor.
+    public OotbRestController() {
+        policy = Sanitizers.FORMATTING;
+    }
+
+    /*
+     * Checks to make sure that the API is running and that there are no issues with
+     * the overrides file. Gets the active profiles and only runs the tests the
+     * active profile relies on the API.
+     */
+
+    @PostMapping(value = "/{activeUserProfile}")
+    public ResponseEntity<String> updateActiveDefaultUser(@PathVariable String activeUserProfile) {
+        logger.info("Entered REST updateActiveUserProfile...");
+        String currentUid = policy.sanitize(userContextService.getCurrentUid());
+        String uri = String.format(API_2_1_BASE + "/ootb/%s", activeUserProfile);
+        ootbAuthenticationFilter.setUserProfile(activeUserProfile);
+        return httpRequestService.makeApiRequest(currentUid, uri, HttpMethod.POST);
```

Comment:
Making a http request to our API side does not look necessary here, completely fine with not returning anything.

Reply:
We are not gonna have any information of multiple active profiles at all in api projeft when api project initiates. It is because for the convenience and flexibility for multiple scenario and don't want to do hard coded whenever something new profile is added or modifed. I just sent the messages to you thru slack about one of the way we can do to figure out this. Also authentication doesn't really affect to the data in groupings project so api side has to be updated